Makefile.include

```diff
@@ -42,6 +42,10 @@ BINDIRBASE := $(abspath $(BINDIRBASE))
 BINDIR ?= $(BINDIRBASE)/$(BOARD)
 BINDIR := $(abspath $(BINDIR))/
 
+ifneq (9, $(words ${RIOTBASE} ${CCACHE_BASEDIR} ${RIOTCPU} ${RIOTBOARD} ${RIOTPROJECT} ${RIOTPKG} ${APPDIR} ${BINDIRBASE} ${BINDIR}))
```

Comment:
My logic was borken. Now I apply each variable individually to `$(words)` and test if any outcome is unequal 1.